TypeScript.py

```diff
@@ -352,6 +352,8 @@ def __init__(self, filename, cc):
         self.changeCountErrReq = -1
         self.lastModChangeCount = cc
         self.modCount = 0
+        self.prevVisStart = -1
+        self.prevHasErrors = False
```

Comment:
The last measured visible region start position and hasErrors flag for the view.